config/environments/development.rb

```diff
@@ -46,7 +46,7 @@
   config.active_record.verbose_query_logs = true
 
   # Raises error for missing translations.
-  # config.action_view.raise_on_missing_translations = true
+  config.action_view.raise_on_missing_translations = ENV.fetch("RAISE_ON_MISSING_TRANSLATIONS", "false") == "true"
```

Comment:
Yep, this is a dev only change that'll let us optionally turn this on via the env variables without making any code changes. It defaults to being off, as it currently stands, but this lets you turn it on if you like. I personally prefer this to always be on, rather than letting it fall back, since it makes sure I don't forget to fill in any translations.